pkg/framework/coremodel/gen.go

```diff
@@ -6,7 +6,7 @@ package main
 
 import (
 	"fmt"
-	"io/ioutil"
+	"io/ioutil" // nolint:depguard
```

Comment:
Because a library expects `[]os.FileInfo` rather than `[]fs.DirEntry`?

Reply:
This could be changed, but since it's behind `//go:build ignore` I'm guessing it's still going to see more work, didn't want to change it around, @sdboyer ?

Reply:
I think it's there because it's a code generator and doesn't need to live inside the application. Sort of like how wire use `wireinject` to find template files and then generate files with `!wireinject` to make sure there's no conflict between the two.

Reply:
I'll patch it up then as well

Reply:
Sorry i missed this last week - yes, this is the right approach. In places where `[]fs.DirEntry` is needed, the appropriate call is `fs.ReadDir`